kernel/dev/vga/graphics.cpp

```diff
@@ -110,7 +148,7 @@ void pixel(uint32_t x, uint32_t y, uint32_t color)
     // Special thanks to the SkiftOS contributors.
     if ((x <= width) && (y <= height))
     {
-        uint8_t *pixel = (uint8_t*)addr + (y * pitch) + (x * pixelwidth);
+        uint8_t *pixel = (uint8_t*)backbuffer+ (y * pitch) + (x * pixelwidth);
```

Comment:
Add a space in between `backbuffer` and `+`.

Reply:
Eagle eyes lol